pkg/perf/healthcheck/healthcheck.go

```diff
@@ -60,21 +62,35 @@ func (h *healthcheckTask) Run(ctx context.Context) (interface{}, error) {
 	log.Debug().Msg("starting health check task")
 	errs := make(map[string][]string)
 
+	cl := perf.GetZbusClient(ctx)
+	zui := stubs.NewZUIStub(cl)
+
 	for label, check := range h.checks {
-		errors := check(ctx)
-		if len(errors) == 0 {
-			continue
+		op := func() error {
+			errors := check(ctx)
+			errs[label] = errorsToStrings(errors)
```

Comment:
why the map? why do you need to use `errs` here this way?

Reply:
errs are returned at the end as the result of the health check task, this is what is stored on redis and responded to the rmb call asking for the health check